src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/WorkaroundsInProcess.cs

```diff
@@ -17,6 +18,17 @@ namespace Roslyn.VisualStudio.IntegrationTests.InProcess
     [TestService]
     internal partial class WorkaroundsInProcess
```

Comment:
Random thought: I love there's a "workarounds" file here so we can easily see where the icky bits are and chase them down long term.